wmn-data.json

```diff
@@ -2069,11 +2111,12 @@
         "name" : "Giphy",
         "uri_check" : "https://giphy.com/channel/{account}",

```

Comment:
I got a redirect from this `uri_check` to `https://giphy.com/{account}` (no channel)

Reply:
As I understand it, there are categories for both "channel" and "no channel" (though I'm not sure of the exact terminology). I've updated the `known` parameter with usernames that are channels. We'll need to create a "No channel" category in the future as well.